poll/public/css/poll.css

```diff
@@ -0,0 +1,218 @@
+    width: 100%;
+    vertical-align: middle;
+    background-color: #fafbfc;
+}
+
+ul.poll-answers, ul.poll-answers-results {
+    list-style-type: none !important;
+    max-width: 80%;
+}
+
+li.poll-answer {
+    display: block;
+    border-bottom-width: .5em;
+    vertical-align: middle;
+    margin-top: 5px;
+    margin-bottom: 5px;
+}
+
+li.poll-spacer {
+    height: .25em;
+}
+
+ul.poll-answers-results {
+    display: table;
+}
+
+li.poll-result {
+    width: 100%;
+    display: table-row;
+    padding-bottom: .2em;
+}
+
+.poll-answer-label {
+    margin-left: .2em;
+    font-weight: bold;
+    display: inline-block;
+    margin-bottom: 5px;
+    margin-top: 5px;
+    white-space: nowrap;
+}
+
+.poll-image {
+    width: 25%;
+    display: inline-block;
+    vertical-align: middle;
+}
+
+.poll-image {
+    margin-left: .5em;
+}
+
+li.poll-result .poll-image {
+    display: table-cell;
+    margin-left: 0;
+}
+
+.poll-image-td {
+    width: 25%;
+}
+
+.poll-image img, .poll-image-td img{
+    width: 100%;
+}
+
+.poll-image-td{
+    display: inline-block;
+}
+
+.poll-percent-container {
+    display: table-cell;
+    text-align: left;
+    padding-left: .2em;
+    vertical-align: middle;
+}
+
+.poll-percent-display {
+    font-weight: bold;
+}
+
+.poll-top-choice {
+    color: #e37222;
+}
+
+.poll-footnote {
+    margin-top: 1em;
+    margin-bottom: 1em;
+    font-size: smaller;
+    font-weight: bold;
+}
+
+.survey-table {
+    width: 100%;
+    border-collapse: collapse;
+    border: 0;
+}
+
+.survey-table td, .survey-table th {
+    border: 1px solid #CCC;
+    border-top: 0;
+}
+
+.survey-table tr:last-child td {
+    border-bottom: 0;
+}
+
+.survey-table tr td:first-child,
+.survey-table tr th:first-child {
+    border-left: 0;
+}
+
+.survey-table tr td:last-child,
+.survey-table tr th:last-child {
+    border-right: 0;
+}
+
+.survey-table thead {
+    border-top: 0;
+    background: none;
+}
+
+.survey-table thead tr th {
+    font-weight: bold;
+    font-size: .9rem;
+}
+
+.survey-table .survey-row {
+    background: none !important;
+}
+
+.survey-table .survey-option {
+    text-align: center;
+    vertical-align: middle;
+}
+
+.survey-option input {
+    margin: 0;
+    padding: 0;
+}
+
+th.survey-answer {
+    text-align: center;
+    width: 7%;
+    line-height: 1em;
+    padding-bottom: .25em;
+}
+
+.poll-header {
+    text-transform: uppercase;
+}
+
+.survey-percentage {
+    font-weight: bolder;
+    font-size: 1.2rem;
+}
+
+.survey-question {
+    font-weight: bold;
+    vertical-align: middle;
+    padding-top: .15em;
+    padding-bottom: .15em;
+}
+
+.survey-choice {
+    background-color: #e5ebee;
+}
+
+/* Counteract Markdown's wrapping in paragraphs */
+.poll-answer p, .survey-question p, .poll-answer-label p{
+    margin: 0;
+    padding: 0;
+}
+
+.poll-voting-thanks span {
+    margin-top: .25em;
+    margin-bottom: .25em;
+    padding: .5em;
+    display: inline-block;
+    background-color: #e5ebee;
+    color: #e37222;
```

Comment:
@sarina I believe this is best addressed using customizable themes, like in mentoring/problem_builder:
* Obtaining theme configuration: https://github.com/open-craft/problem-builder/blob/master/problem_builder/mentoring.py#L208-219
* Adding a theme to rendered fragment: https://github.com/open-craft/problem-builder/blob/master/problem_builder/mentoring.py#L268-272

Default theme will than contain proper (from a11y point of view) colors, and there would be an alternative theme containing this particular config, as it is used by third party client.